inc/template.php

```diff
@@ -1639,6 +1639,11 @@ function tpl_actiondropdown($empty = '', $button = '&gt;') {
         'user_tools' => array('login', 'register', 'profile', 'admin'),
     );
 
+    $evt = new Doku_Event('TEMPLATE_MOBILETOOLS_DISPLAY', $action_structure);
+    if ($evt->advise_before()) {
+        $action_structure = $evt->data;
+    }
```

Comment:
This if should wrap around the whole output. That way plugins could supress the output and do their own thing.